actions/user/changepassword.php

```diff
@@ -25,7 +25,13 @@
 
 if (execute_new_password_request($user_guid, $code, $password)) {
 	system_message(elgg_echo('user:password:success'));
-	login(get_entity($user_guid));
+	
+	try {
+		login(get_entity($user_guid));
+	} catch (LoginException $e) {
+		register_error($e->getMessage());
```

Comment:
it's the same as login (that's where i copied it from)
https://github.com/Elgg/Elgg/blob/master/actions/login.php#L51-L58